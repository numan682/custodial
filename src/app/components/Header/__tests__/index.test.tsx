import React from 'react'
import { render, screen } from '@testing-library/react'

import { Header, ModalHeader } from '..'

describe('<Header />', () => {
  it('should render component', () => {
    const { container } = render(<Header>Title</Header>)

    expect(container).toMatchSnapshot()
  })

  it('should render header with text align property', () => {
    render(<Header textAlign="center">Title</Header>)

    expect(screen.queryByRole('heading')).toHaveStyle('text-align: center')
  })
})

describe('<ModalHeader />', () => {
  it('should render component', () => {
    const { container } = render(<ModalHeader>Title</ModalHeader>)

    expect(container).toMatchSnapshot()
  })
})
