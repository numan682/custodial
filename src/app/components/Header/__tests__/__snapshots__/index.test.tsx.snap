// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render component 1`] = `
.c0 {
  margin-top: 0px;
  font-size: 34px;
  line-height: 40px;
  max-width: 816px;
  font-weight: 600;
  overflow-wrap: break-word;
}

@media only screen and (max-width:768px) {
  .c0 {
    margin-top: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c0 {
    font-size: 26px;
    line-height: 32px;
    max-width: 624px;
  }
}

<div>
  <h1
    class="c0"
  >
    Title
  </h1>
</div>
`;

exports[`<ModalHeader /> should render component 1`] = `
.c0 {
  margin-top: 0px;
  font-size: 34px;
  line-height: 40px;
  max-width: 816px;
  font-weight: 600;
  overflow-wrap: break-word;
}

@media only screen and (max-width:768px) {
  .c0 {
    margin-top: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c0 {
    font-size: 26px;
    line-height: 32px;
    max-width: 624px;
  }
}

<div>
  <h2
    class="c0"
  >
    Title
  </h2>
</div>
`;
